projects/swc-starter-webpack/manifest.json

```diff
@@ -9,27 +9,27 @@
     },
     "entrypoints": [
         {
-          "type": "panel",
-          "id": "vanilla",
-          "minimumSize": {
-            "width": 230,
-            "height": 200
-          },
-          "maximumSize": {
-            "width": 2000,
-            "height": 2000
-          },
-          "preferredDockedSize": {
-            "width": 230,
-            "height": 300
-          },
-          "preferredFloatingSize": {
-            "width": 230,
-            "height": 300
-          },
-          "label": {
-            "default": "Starter Panel"
-          }
+            "type": "panel",
```

Comment:
this format is already added in the menu PR.

Reply:
let it be, which ever PR gets merged first, for the subsequent one, i will resolve it.